mock/cpp_mock_scenarios/src/traffic_light/phase_control.cpp

```diff
@@ -30,10 +29,7 @@ class PhaseControlScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit PhaseControlScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "phase_control",
-      ament_index_cpp::get_package_share_directory("cargo_delivery") + "/maps/kashiwa",
```

Comment:
.osm file should be exactly fixed in scenario.
So, please specify it in constructor.